packages/gatsby-core-utils/src/fetch-remote-file.ts

```diff
@@ -72,9 +74,63 @@ const ERROR_CODES_TO_RETRY = [
   `ERR_GOT_REQUEST_ERROR`,
 ]
 
+/********************
+ * Queue Management *
+ ********************/
+
+const GATSBY_CONCURRENT_DOWNLOAD = process.env.GATSBY_CONCURRENT_DOWNLOAD
+  ? parseInt(process.env.GATSBY_CONCURRENT_DOWNLOAD, 10) || 0
+  : 200
+
+const q: queue<IFetchRemoteFileOptions> = Queue(
+  pushToQueue,
+  GATSBY_CONCURRENT_DOWNLOAD
+)
+
+/**
+ * pushToQueue
```

Comment:
I found this naming very confusing.

I'd vote to rename this to `worker` as in https://github.com/mcollina/fastq#using-with-typescript-callback-api